CONTRIBUTING.md

```diff
@@ -28,6 +28,13 @@ commit message:
 $ git commit -s -m 'This is my commit message'
 ```
 
+If you have already made a commit and forgot to include the sign-off, you can amend your last commit
+to add the sign-off with the following command, which can then be force pushed.
+
+```
+git commit --amend --signoff
```

Comment:
The above command uses the short hand flag `-s`. Should the long form then be used here or the other example(s) update to the long form?